src/mixxx.cpp

```diff
@@ -598,6 +596,11 @@ void MixxxMainWindow::initializeWindow() {
     Pal.setColor(QPalette::Background, MenuBarBackground);
     m_pMenuBar->setPalette(Pal);
 
+    // restore window size/position
+    QSettings settings("mixxxdj", "mixxx");
```

Comment:
Code is cheaper than state -- I'd rather write a few more lines of code than have two separate configuration files in every users profile. What happens when we want to migrate to a new profile format a year down the line? We'll have to migrate the user data from N+1 files instead of N. 

Take a look at the horror that is our version upgrade code:
https://github.com/mixxxdj/mixxx/blob/master/src/preferences/upgrade.cpp
The less user state we have to deal with the better. 


BTW, here's what the wrapper code would look like:
```c++
m_pSettingsManager->settings()->set(
    ConfigKey("[MainWindow]", "geometry"), 
    saveGeometry().toByteArray().toBase64());
```